### DISCUSSION TITLE
Changing filename on only case/capitalisation adjustment

### DISCUSSION THREAD

**Discussion by cjproc** [gsantner/markor#1542] [Q&A] [answered]
## General information
* App version: 2.8.1
* Android version: 10
* Device: Nexus 5X

### Description

Recreate:
1. Open "Files"  
2. Highlight a file  
3. Tap to change name  
4. Vary the case of one of the letters
5. "File/folder already exists! Please use a different name"

**Comment by cjproc** [comment]
Just tried creating two files "test.md", "Test.md" using filesystem app, but that's not allowed.

Seems like a feature of the filesystem, rather than the app. Sorry! I'll close it.

**Comment by gsantner** [comment, marked as answer]
Yes filesystem on android is usually insensitive.

Simple trick: rename test.md to xest.md, and then to Test.md.

**Reply by aisbergde** [reply]
it would be helpful to have an additional action when renaming a file:

* rename filename to lower case

I try to use only lower case, but the keyboard often changed some letters (especially at the beginning of a word) to upper case.

**Reply by harshad1** [reply]
IMO adding markor features to work around your keyboard settings is not a good idea

**Reply by aisbergde** [reply]
In General I am happy with my keyboard settings. The only time, when I would prefer to use only lower letters, is when I create files with Markor. I have no idea how to set specific keyboard settings per app.

**Reply by aisbergde** [reply]
And the request was not only about the creation process, but about maintenance of existing filenames.
